addons/account/models/chart_template.py

```diff
@@ -159,8 +159,8 @@ def _notify_accountant_managers(taxes_to_check):
             )
 
     env = api.Environment(cr, SUPERUSER_ID, {})
-    chart_template_id = env['ir.model.data']._xmlid_to_res_id(chart_template_xmlid)
-    companies = env['res.company'].search([('chart_template_id', '=', chart_template_id)])
+    chart_template_id = env['ir.model.data'].xmlid_to_res_id(chart_template_xmlid)
```

Comment:
Hi @dylankiss , I was browsing some code and found your issue. It's a missing underscore.
`def _xmlid_to_res_id(self, xmlid, raise_if_not_found=False):`

But I think what you really need might be `self.env.ref(xml_id)` instead.
I think that `_xmlid_to_res_id` the low level version of `ref` which would mean we aren't suppose to use it directly.

Reply:
@Levizar I saw it last night. Just updated the PR. Something I missed in the conflict of the FW port. I changed it to use the `env.ref` method as you suggest, since it's not using a private method this way.